.fixtures-latest.yml

```diff
@@ -17,6 +17,12 @@ fixtures:
     yumrepo_core:
       repo: https://github.com/puppetlabs/puppetlabs-yumrepo_core.git
       puppet_version: ">= 6.0.0"
+    augeas_core:
+      repo: https://github.com/puppetlabs/puppetlabs-augeas_core.git
+      puppet_version: ">= 6.0.0"
+    yumrepo_core:
```

Comment:
yumrepo_core is already in this file, so this addition is a duplicate of whats a few lines higher.

Reply:
Ok, thanks for the detailed pointers to multi-container tests. I'll look into it. The other fixes you proposed also make sense, will look into them as well.